pages/docs/reference/events/send.mdx

```diff
@@ -25,9 +25,7 @@ To send events from within of the context of a function, use [`step.sendEvent()`
 ---
 
 
-## `inngest.send(eventPayload | eventPayload[]): Promise`
-{/* TODO - Show the event ids in the return value when we add to the SDK */}
-{/*## `inngest.send(eventPayload | eventPayload[]): Promise<string[]>`*/}
+## `inngest.send(eventPayload | eventPayload[]): Promise<{ ids: String[] }>`
```

Comment:
```suggestion
## `inngest.send(eventPayload | eventPayload[]): Promise<{ ids: string[] }>`
```

Reply:
Writing too much Kotlin recently 🤦

Reply:
Side note - this notation is kind of half JS / half TS - we probably should improve that over time